demo/docs/features/measurements.md

```diff
@@ -0,0 +1,55 @@
+# Measurements
+
+Measurements (aka metrics) are indicators for tracking mainly the performance and resource consumption but they can be
```

Comment:
Can cut out some adverbs to be more succinct. Something like "Measurements (aka metrics) are indicators for tracking performance or other numeric values."